test/summary/clearlydefinedTests.js

```diff
@@ -687,17 +690,44 @@ describe('ClearlyDescribedSummarizer addSourceArchiveData', () => {
       summarizer.addSourceArchiveData(
         result,
         { manifest: { summary: { project: { licenses: [{ license: { url } }] } } } },
-        testCoordinates
+        sourceArchiveTestCoordinates
       )
       if (data[url])
         assert.deepEqual(result, {
-          ...expectedResult
+          ...expectedResult,
+          licensed: { declared: data[url] }
         })
       else assert.deepEqual(result, expectedResult)
     }
   })
-})
+  it('should set the correct urls and no license when data section is empty', () => {
+    let result = {}
+    summarizer.addSourceArchiveData(result, {}, testCoordinates)
+    assert.deepEqual(result, expectedResult)
+  })
+
+  it('should set declared license from manifest license name and URL', () => {
+    const data = new Map([
+      [[{ license: [{ name: ['Apache-2.0'], url: ['https://opensource.org/licenses/Apache-2.0'] }] }], 'Apache-2.0'],
+      [[{ license: [{ name: ['The MIT License (MIT)'], url: ['http://opensource.org/licenses/MIT'] }] }], 'MIT'],
+      [[{ license: [{ name: ['LGPL 2.1'], url: ['https://www.gnu.org/licenses/lgpl-2.1.html'] }, { name: ['Apache License 2.0'], url: ['https://www.apache.org/licenses/LICENSE-2.0'] }] }], 'LGPL-2.1 OR Apache-2.0'],
+      [[{ license: [{ name: ['CDDL + GPLv2 with classpath exception'], url: ['https://oss.oracle.com/licenses/CDDL+GPL-1.1'] }] }], 'NOASSERTION']
+    ])
 
+    data.forEach((expected, licenses) => {
+      let result = {}
+      summarizer.addSourceArchiveData(result, { manifest: { summary: { licenses } } }, sourceArchiveTestCoordinates)
+      if (expected)
+        assert.deepEqual(result, {
+          described: {
+            urls: expectedUrls
+          },
+          licensed: { declared: expected }
+        })
```

Comment:
assert.deepEqual(result, { ...expectedResult, licensed: { declared: expected }}?

Reply:
The formatting is being changed by prettier when i run command `npm run lint:fix`
Please let me know if there is any way I can fix this. Otherwise I will get error in checks.